### PR TITLE
fixed flicker in EmbeddedFrame component

### PR DESCRIPTION
**Pull request by LemonFace0309**
https://www.loom.com/share/fe733c5bfe7e465b9f56fff851155288?sid=68a48078-4c0b-4c01-92db-d2cb907210a2

### CODE REVIEW

**Comment by LemonFace0309** [comment]
* **#34** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/34?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @LemonFace0309 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: components/SyncCreationWizard.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    refetchSyncManagementLinks,
+    workspaceAccessToken,
+  )
+
+  const linkWithSourcePrepopulated =
+    syncManagementLink.uri + "&form_connection_id=" + sourceId + "&form_source_type=warehouse"
+
+  return (
+    <EmbeddedFrame
+      connectLink={linkWithSourcePrepopulated}
```
Comment (LemonFace0309) [inline components/SyncCreationWizard.js:24]: @yeldah3 changed this to be a constant value instead of a function for simplicity

**Review by yeldah3 — APPROVED** [review]
Yay! I want a React lesson in this tomorrow pls. Is it basically that SyncManagement is still re-rendering on button clicks, but SyncCreationWizard is now stable since its props have been set to things that wont change by clicking things in the parent? (Ie running/pausing/deleting syncs?)

**Comment by LemonFace0309** [comment]
> Yay! I want a React lesson in this tomorrow pls. Is it basically that SyncManagement is still re-rendering on button clicks, but SyncCreationWizard is now stable since its props have been set to things that wont change by clicking things in the parent? (Ie running/pausing/deleting syncs?)

I believe it's because `SyncCreationWizard` was defined in `SyncManagement` which was constantly being rerendered when the props changed, causing `SyncCreationWizard` to be instantiated again and again

**Pull request merged**